examples/random-go/go.mod

```diff
@@ -0,0 +1,10 @@
+module goaws
+
+go 1.13
+
+require (
+	github.com/pulumi/pulumi-command/sdk/v3 v3.0.1
```

Comment:
I am unsure where the `v3` is coming from.  Will want to fix that before releasing to ensure the Go package isn't accidentally being marked as 3.0.  I expect this is an issue due to using an old version of codegen.